sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa

```diff
@@ -1,107 +0,0 @@
-#!/usr/bin/pulseaudio -nF
-#
-# This file is part of PulseAudio.
-#
-# PulseAudio is free software; you can redistribute it and/or modify it
-# under the terms of the GNU Lesser General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
-# (at your option) any later version.
-#
-# PulseAudio is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-# General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public License
-# along with PulseAudio; if not, write to the Free Software Foundation,
-# Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
-
-# This startup script is used only if PulseAudio is started per-user
-# (i.e. not in system mode)
-
-.nofail
-
-.fail
-
-load-module module-droid-keepalive
-
-load-module module-meego-parameters cache=1 directory=/var/lib/nemo-pulseaudio-parameters use_voice=false
-load-module module-meego-mainvolume virtual_stream=true
-
-### Automatically restore the volume of streams
-load-module module-stream-restore-nemo restore_device=no restore_volume=yes restore_muted=no route_table=/etc/pulse/x-maemo-route.table fallback_table=/etc/pulse/x-maemo-stream-restore.table use_voice=false sink_volume_table=/etc/pulse/x-maemo-sink-volume.table
-
-load-module module-match table=/etc/pulse/x-maemo-match.table key=application.name
-
-### Automatically augment property information from .desktop files
-### stored in /usr/share/application
-load-module module-augment-properties
-
-load-module module-null-sink sink_name=sink.null rate=48000
-
-load-module module-droid-card rate=48000 mute_routing_before=24576 mute_routing_after=4096
-load-module module-droid-glue
-
-load-module module-null-sink sink_name=sink.fake.sco rate=8000 channels=1
-load-module module-null-source source_name=source.fake.sco rate=8000 channels=1
-load-module module-bluez4-discover sco_sink=sink.fake.sco sco_source=source.fake.sco
-
-load-module module-policy-enforcement route_sources_first=true
```

Comment:
We had to add this file for the route_sources_first=true to get the radio working. So we need this file. If there are any updates to the file provided by upstream, we could probably merge them into ours?